### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by VladyslavSamkov**
Homework 1

### CODE REVIEW

File: cypress/e2e/sign_in.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import {sigh_in_selectors as sigh_in_selectors} from "../selectors/sigh_in_selectors";
+
+describe('UI tests for sign in page',
+    () => {
```
Comment (ievgendco) [inline cypress/e2e/sign_in.js]: Function should be in the same line with the describe

File: cypress/e2e/sign_in.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import {sigh_in_selectors as sigh_in_selectors} from "../selectors/sigh_in_selectors";
```
Comment (ievgendco) [inline cypress/e2e/sign_in.js]: spaces should be added inside of the name exported item like:
import { sigh_in_selectors } as sigh_in_selectors from "../selectors/sigh_in_selectors";

**Review by ievgendco — CHANGES_REQUESTED** [review]

File: cypress/e2e/sign_in.js (hunk 1 above)
Comment (VladyslavSamkov) [reply cypress/e2e/sign_in.js]: Moved function to the same like with the describe, please review

File: cypress/e2e/sign_in.js (hunk 2 above)
Comment (VladyslavSamkov) [reply cypress/e2e/sign_in.js]: Added spaces inside of the exported name, please review

File: cypress/e2e/sign_in_page.spec.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                .and('have.css', 'color', 'rgb(244, 67, 54)');
+        })
+
+        it('4.2 Error message not appears', () => {
+            cy.get(sign_in_selectors.username_field).type('Hello World!');
+            cy.get(sign_in_selectors.password_field).click();
+            cy.get(sign_in_selectors.username_helper_text).should('not.exist');
+        })
+
+        it('5.1 Remember me disabled by default', () => {
```
Comment (anatoly-devico) [inline cypress/e2e/sign_in_page.spec.js:46]: in test suite should be only tests which are listed here:
https://github.com/anatoly-devico/cypress_devico_courses/blob/main/cypress/e2e/authentication.spec.js

File: .idea/workspace.xml [hunk 4]
```diff
@@ -1,17 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ChangeListManager">
-    <list default="true" id="9e5680cd-8407-4143-bec1-26afffb8c9b5" name="Changes" comment="wer">
+    <list default="true" id="9e5680cd-8407-4143-bec1-26afffb8c9b5" name="Changes" comment="1. Added spaces inside of the name exported.&#10;2. Moved function to the same line describe.">
       <change beforePath="$PROJECT_DIR$/.idea/workspace.xml" beforeDir="false" afterPath="$PROJECT_DIR$/.idea/workspace.xml" afterDir="false" />
```
Comment (anatoly-devico) [inline .idea/workspace.xml]: Don't push this file

**Review by anatoly-devico — CHANGES_REQUESTED** [review]

File: .idea/workspace.xml (hunk 4 above)
Comment (VladyslavSamkov) [reply .idea/workspace.xml]: Resolved in https://github.com/VladyslavSamkov/vladyslav_samkov_cypress/pull/6/files

**Pull request merged**